src/dotfilepatcher.cpp

```diff
@@ -111,7 +111,7 @@ static const char svgZoomFooter2[] = R"svg(
 [data-mouse-over-selected='true']  { opacity: 1.0; }
 ]]>
 </style>
-<script><![CDATA[
+<script xlink:type="simple"><![CDATA[
```

Comment:
where did you find that xlink:type is more standard conforming? I found [here](https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/xlink:type) it is actually a deprecated tag.

Reply:
I found it in the svg definitions. It is possible that the `xlink:type` is deprecation for the svg 2.0 version but doxygen is using svg 1.1  (as is e.g. `dot`).

Reply:
I read [here](https://www.w3.org/TR/SVG11/linking.html) that: "In SVG 1.1, only simple links are available. Links are simple links by default, so the attribute xlink:type="simple" is optional and may be omitted on simple links. ". So I think it is better to leave out the xlink:type="simple" (smaller file size).

Reply:
Yes only the value "simple" is possible, though when checking against the dtd with xmllint I get a warning.
In the dtd I fined everywhere. n respect to title, %XLINK.pfx;type ( simple ) #FIXED 'simple' so because of the warning I added the attribute.